docs/api/v2/node.md

```diff
@@ -22,7 +22,7 @@ proc init*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   ##
   ## Status: Implemented.
 
-proc start*(node: WakuNode) {.async.} =
+proc start*(node: WakuNode, topics = newSeq[string]()) {.async.} =
```

Comment:
Explain what topics is below.

Reply:
As in, explain the semantics.

As a user, I want to be able to read the signature and docstring and know what it is doing without knowing anything else.